packages/devtools-solana/src/omnigraph/sdk.ts

```diff
@@ -23,10 +25,16 @@ export abstract class OmniSDK implements IOmniSDK {
         return `Solana program @ ${formatOmniPoint(this.point)}`
     }
 
-    protected createTransaction(data: string): OmniTransaction {
+    protected async createTransaction(transaction: Transaction): Promise<OmniTransaction> {
+        const { blockhash } = await this.connection.getLatestBlockhash('finalized')
```

Comment:
no change needed: perhaps this should be placed into a variable to avoid magic strings -- not preventing from approval bc I see it is not used elsewhere